ec2/spark_ec2.py

```diff
@@ -872,9 +890,16 @@ def deploy_files(conn, root_dir, opts, master_nodes, slave_nodes, modules):
     if "." in opts.spark_version:
         # Pre-built Spark deploy
         spark_v = get_validate_spark_version(opts.spark_version, opts.spark_git_repo)
+        tachyon_v = get_tachyon_version(spark_v)
     else:
         # Spark-only custom deploy
         spark_v = "%s|%s" % (opts.spark_git_repo, opts.spark_version)
+        if opts.tachyon_version is None:
```

Comment:
@shivaram You mean for advanced users, let them manually specify tachyon_v for themselves?

or `export TACHYON_VERSION=xxx` manually in bash before running `spark_ec2`, but then we have to disable `export TACHYON_VERSION=xx` in `spark_ec2.py`. 

I think this option is appropriate, first, for users who do not use git hash to deploy, nothing has changed in the CLI, second, for those use git hash, this option is handy, just need to update document.